### PR TITLE
Big moves

### PR DESCRIPTION
**Pull request by sunniraleigh**
- fix syntax error on pricing page
- add email to link to footer
- update readme
- make login and signup form consistent styles
- add more station info to map markers
- add button to pricing page from map page

### CODE REVIEW

**Review by amarrap — APPROVED** [review]
Looks good!

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -1,38 +1,32 @@
...
-          <br>
+      <div class="card-body">
 
-          <!-- Username input -->
-          <div class="form-outline mb-4">
-            <%= f.text_field :username, class: "form-control", id: "form-login-username" %>
-            <%= f.label :username, class: "form-label", for: "form-login-username"%>
-          </div>
+        <!-- Username input -->
+        <div class="form-outline mb-4">
```
Comment (luciebrock) [inline app/views/sessions/new.html.erb:10]: awesome

File: db/migrate/20210830174339_create_stations.rb [hunk 2]
```diff
@@ -4,6 +4,10 @@ def change
       t.integer :identifier
       t.string  :name
       t.string  :address
+      t.boolean :has_kiosk
```
Comment (luciebrock) [inline db/migrate/20210830174339_create_stations.rb:7]: awesomeeeee

**Review by luciebrock — APPROVED** [review]
so awesome. love it

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -67,6 +67,10 @@
       </main>
 
       <footer>
+      <div>
```
Comment (epartakki) [inline app/views/layouts/application.html.erb:70]: This is a lie

File: README.md [hunk 4]
```diff
@@ -3,66 +3,103 @@
 Smith College CSC223: Software Engineering
 Starter App for ValetBike project
 
-### Overview of dev environment setup (e.g. updates to _env.rb)
+## Overview of dev environment setup (e.g. updates to _env.rb)
```
Comment (epartakki) [inline README.md:6]: thank you for sorting this!

**Review by epartakki — APPROVED** [review]

**Pull request merged**